lib/view_model/migratable_view.rb

```diff
@@ -19,12 +19,14 @@ def initialize_as_migratable_view
       @migrations_lock   = Monitor.new
       @migration_classes = {}
       @migration_paths   = {}
-      @realized_migration_paths = true
+      @previous_names    = {}
+      @realized_paths    = true
+      @previous_name_cache = nil
```

Comment:
This has unfortunate overlap
 - `previous_names` = points at which the name changed, very internal
 - `previous_names_cache` = all valid names, aka "versioned view names"
```suggestion
      @versioned_view_names = nil
```

And similarly change `cache_previous_names!` to `cache_versioned_view_names!`.

Reply:
Makes sense, I'll do that.